### ISSUE TITLE
API for triggering a close request

### ISSUE THREAD

**Issue by lukewarlow** [w3c/webdriver#1827]
HTML recently added the concept of ["Close Requests"](https://html.spec.whatwg.org/multipage/interaction.html#close-requests), and ["CloseWatchers"](https://html.spec.whatwg.org/multipage/interaction.html#the-closewatcher-interface).

It would be good if WebDriver had a way to trigger a close request that was independent of a specific input type. The WPT tests were intentionally setup in such a way that a WebDriver close request API could be used in future with a minor change.

**Comment by whimboo** [comment]
If we could benefit from such a feature we might want to have it for WebDriver BiDi instead?

CC @jgraham for his feedback.

**Comment by jgraham** [comment]
Yes, agreed. New features are generally happening in BiDi, so I'd suggest we consider adding it there.
